blockstore_proxy.go

```diff
@@ -25,8 +25,7 @@ import (
 const (
 	EnvProxyGateway = "PROXY_GATEWAY_URL"
 
-	DefaultProxyGateway = "http://127.0.0.1:8080"
-	DefaultKuboRPC      = "http://127.0.0.1:5001"
+	DefaultKuboRPC = "http://127.0.0.1:5001"
```

Comment:
ℹ️ `DefaultProxyGateway` was not being used and is no longer the default, since it must be set. I also updated the documentation to reflect that. This has been the case since f78b4ef44a6fa751364fe4a63ba511ccfb049600.

Reply:
@hacdias thoughts on also removing  `DefaultKuboRPC` and when `KUBO_RPC_URL` is not explicitly set we [dont register listeners for /api/v0 at all here](https://github.com/ipfs/bifrost-gateway/blob/b852ebadb04011684b935a949c50de1e4c86e6c6/handlers.go#L166-L169)?

Rhea does not use `DefaultKuboRPC`, instead it sets explicit `KUBO_RPC_URL`, and we don't care about `/api/v0` outside of Rhea tbh.